raiden/network/transport/matrix/transport.py

```diff
@@ -997,7 +993,7 @@ def _get_room_for_address(
         if self._stop_event.ready():
             return None
         address_hex = to_normalized_address(address)
-        assert address and address in self._address_to_userids,\
+        assert address and self._address_mgr.is_address_known(address),\
```

Comment:
Could you please separate the assert message into `msg` and get rid of the `\` in this line?